### PR TITLE
[QOLOE-216] Banner design QA

### PR DESCRIPTION
**Pull request by elvishu**
- [QOLOE-216] Banner design QA
- Hotfix for medium viewport width in Storybook
- Fix for breadcrumb's active item colour

[QOLOE-216]: https://ssq-qol.atlassian.net/browse/QOLOE-216?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/bs5/breadcrumbs/breadcrumbs.scss
Comment (elvishu) [inline src/components/bs5/breadcrumbs/breadcrumbs.scss:1]: Updated the colour for active item in breadcrumb - dark & dark-alt theme.

File: src/components/bs5/banner/banner.scss
Comment (elvishu) [inline src/components/bs5/banner/banner.scss:1]: - Design QA fixes
- Update variables name to use prefix

File: src/components/bs5/banner/banner.hbs
Comment (elvishu) [inline src/components/bs5/banner/banner.hbs:1]: Added btn-toolbar to make CTA buttons responsive and padded nicely - similar with Button component.
Update variable following scss variables name update.

File: .storybook/preview.js
Comment (elvishu) [inline .storybook/preview.js:1]: Updated medium viewport width in Storybook to 700px, following the same viewport settings in CSS (qld-variables.scss).

**Review by stvp-qld — APPROVED** [review]

**Review by ghazal-salehi — APPROVED** [review]

**Comment by Epinder-SSQ** [comment]
@re-becca UI reviewed and cross-checked on chromatic with design QA, no issues found or changes required.

**Review by Epinder-SSQ — APPROVED** [review]

**Pull request merged**